package.json

```diff
@@ -7,7 +7,7 @@
 		"loader-utils": "^1.0.2"
 	},
 	"peerDependencies": {
-		"coffee-script": "1.x"
+	    "coffeescript": "*"

```

Comment:
The risk with this is that if they break their API in a major, the loader will break. So change to a range if you want to be careful.